lib/src/settings_list.dart

```diff
@@ -23,30 +23,32 @@ class SettingsList extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      color: Theme.of(context).brightness == Brightness.light
-          ? backgroundColor ?? lightBackgroundColor ?? backgroundGray
-          : backgroundColor ?? darkBackgroundColor ?? Colors.black,
-      child: ListView.builder(
-        physics: physics,
-        shrinkWrap: shrinkWrap,
-        itemCount: sections.length,
-        itemBuilder: (context, index) {
-          AbstractSection current = sections[index];
-          AbstractSection futureOne;
-          try {
-            futureOne = sections[index + 1];
-          } catch (e) {}
+    return Material(
+      child: Ink(
```

Comment:
Wrapping the `ListView` in an `Ink` will cause the splash to be applied all across the list as opposed to just a single tile.